static/ca_bydate.js

```diff
@@ -173,9 +173,11 @@ function redoFilter() {
   table.clearFilter();
   if (countySel.value) {
     table.addFilter("county", "=", countySel.value);
+    setSubHeader = document.getElementById("sub-header").innerHTML = countySel.value;
   }
   if (locationTypeSel.value) {
     table.addFilter("location_type", "=", locationTypeSel.value);
+    setSubHeader = document.getElementById("sub-header").innerHTML = locationTypeSel.value;
```

Comment:
this too

Reply:
We use these lines of code to show the name of the specific county or venue that the user selects as a header above the chart of both of the chart pages